components/bms_worker/src/Environment.c

```diff
@@ -176,22 +176,22 @@ static void Environment10Hz_PRD()
     {
         if (SHT_getData())
         {
-            ENV.values.board.ambient_temp = sht_chip.data.temp;
-            ENV.values.board.rh           = sht_chip.data.rh;
```

Comment:
same for these two but for the file HW_SHT40.h (only marked in the header file, not in the c file).